src/page/form/Form.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react'
+
+const subject = [
+  "Đá bóng",
+  "Cầu lông",
+  "Đua thuyền"
+];
+
+export default class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      txtname:"",
+      txtemail:"",
+      txtpass:"",
+      txtavatar:"",
+      txtphone:"",
+      txtage:1,
+      txtgender:"male",
+      txtnote:"Buổi học đầu tiên về HTML",
+      txthobbi: new Set()
+    };
+    this.handleChange =this.handleChange.bind(this);
+    this.handleSubmit =this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    var target = event.target;
+    var name = target.name;
+    var value = target.type === 'checkbox' ? target.checked : target.value;
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    var nhobbi = "";
+    for(const hb of this.state.txthobbi){
```

Comment:
Chỗ cần dùng const thì không dùng, chỗ không cần dùng thì lại dùng
```suggestion
    for(let hb of this.state.txthobbi) {
```